iXBRLViewerPlugin/viewer/src/js/index.js

```diff
@@ -51,33 +50,55 @@ function getTaxonomyData() {
     return null;
 }
 
+function checkDocumentSetBrowserSupport() {
+    if (document.location.protocol == 'file:' && window.chrome) {
```

Comment:
Why does Chrome not support this?

Reply:
Chrome is very strict about loading local resources (even more strict than Firefox).  I don't know if it's the same issue, but [this bug](https://bugs.chromium.org/p/chromium/issues/detail?id=40787) has some details about AJAX.

Reply:
The viewer relies on running JS across iframes (i.e. the parent document modifies the contents of the iframes).  This is OK if the iframes are the same origin, but Chrome treats a pair of file URLs as cross-origin.  In single document mode, the iframe is populated dynamically from the same document, which apparently is OK.

We discussed adding a mode to the plugin to have it generate the viewer as a single (potentially very large) HTML file, and then populating all iframes dynamically.